python/pydantic_core/_pydantic_core.pyi

```diff
@@ -884,14 +1001,36 @@ def list_all_errors() -> list[ErrorTypeInfo]:
     """
 @final
 class TzInfo(datetime.tzinfo):
+    """A `pydantic-core` implementation of the `datetime.tzinfo` class.
+
+    Docstrings for attributes sourced from the base class, [`datetime.tzinfo`](https://docs.python.org/3/library/datetime.html#datetime.tzinfo)."""
```

Comment:
Maybe this should be a `#` comment instead, probably doesn't need to be exposed to users?